src/templates/next/components/Siderail/Siderail.tsx

```diff
@@ -0,0 +1,129 @@
+import { BiLeftArrowAlt } from "react-icons/bi"
+import { SiderailProps } from "~/common"
+import { Heading } from "../../typography/Heading"
+import { Paragraph } from "../../typography/Paragraph"
+import { useState } from "react"
+import { MdKeyboardArrowUp, MdKeyboardArrowDown } from "react-icons/md"
+
+const SiderailMobile = ({
+  parentTitle,
+  parentUrl,
+  items,
+  LinkComponent = "a",
+}: Omit<SiderailProps, "type">) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  return (
+    <div className="lg:hidden">
+      <button
+        className="w-full flex gap-2 px-5 py-4 justify-between items-center border-b-2 border-black"
+        onClick={() => setIsExpanded(!isExpanded)}
+      >
```

Comment:
nit: can put an aria-label for this + adapt the label to the state

Reply:
[aaf094a](https://github.com/isomerpages/isomer-components/pull/43/commits/aaf094aee6033ad29981e5fd3debb4ee47d0c878)

don't rly know what to call this cus I saw that you already used the phrase `Close navigation menu` for the navbar